python/eos/analysis.py

```diff
@@ -22,6 +22,9 @@
 import scipy
 import pypmc
 
+from eos import error, info, warn
```

Comment:
The rest of the code uses ``eos.error``, etc. Can you please convert your uses and removes this line?